cfme/utils/browser.py

```diff
@@ -232,7 +232,7 @@ def __init__(self, browser_factory):
         self._browser_renew_thread = None
 
     def coerce_url_key(self, key):
-        return key or store.base_url
+        return key or store.current_appliance.url
```

Comment:
**OPTIONAL** lets add a note here to eventually sort out that reliance on the current appliance

Reply:
Added TODO note